src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -22,19 +22,19 @@
           UUID.randomUUID(),
           "test_username_one",
           "$2a$10$/zf4WlT2Z6tB5sULB9Wec.QQdawmF0f1SbqBw5EeJg5uoVpKFFXAa",
-          Instant.ofEpochMilli(1000), "random bio 5EeJg5uoVpK");
+          Instant.ofEpochMilli(1000), "random bio 5EeJg5uoVpK", "English");
   private final User USER_TWO =
       new User(
           UUID.randomUUID(),
           "test_username_two",
           "$2a$10$lgZSbmcYyyC7bETcMo/O1uUltWYDK3DW1lrEjCumOE1u8QPMlzNVy",
-          Instant.ofEpochMilli(2000), "random bio C7bETcMo/O1uUltWYD");
+          Instant.ofEpochMilli(2000), "random bio C7bETcMo/O1uUltWYD", "Spanish");
```

Comment:
Do languages need to be entered in English?

Reply:
Oh this was just for testing, we will end up using language codes.